UnitTests/TestDynamicResponse.cs

```diff
@@ -86,17 +86,6 @@ public async Task RuntimeErrorsAreThrown()
             Assert.NotNull(ex);
         }
 
-#if NET462
-        [Fact]
-        public async Task RuntimeErrorsIncludeLineNumber()

```

Comment:
Sannsynligvis har .NET også mulighet for å få linjenummer i stacktrace. Sjekk argumenter til `CSharpScript.Create`. Se https://github.com/dotnet/roslyn/issues/13482

Reply:
Takk.

Fixed: https://github.com/codeape2/netmockery/pull/22/commits/2d22920e0211a11e61456f778369d7cb2e837dbf